wiki/www/contributions.py

```diff
@@ -56,15 +56,6 @@ def get_context(context):
 @frappe.whitelist()
 def get_contributions(limit):
 	context = frappe._dict()
-	context.no_cache = 1
-	context.no_sidebar = 1
```

Comment:
no clue why this was added - isn't even used on the client side 🤔 

https://github.com/frappe/wiki/blob/8f324caf522db16536c83a93ee70b5a0cdd8bceb/wiki/www/contributions.html#L78-L95

Reply:
To append contributions in the list if More button is click

Reply:
Ah I was talking about the nocache and nosidebar keys 😅

Since this is a request call, these keys are not really helpful (plus they're not used on the client side as well) ..probably they must've been copy-pasted from the get_context function 😅

Reply:
remove them👍🏻

Reply:
Already removed 😀